src/components/services/WalletConnectService.tsx

```diff
@@ -99,7 +99,7 @@ export const WalletConnectService = (props: Props) => {
 }
 
 // Cleaners
-const payloadAmounts = asObject({ nativeAmount: asString, networkFee: asString, tokenId: asOptional(asString) })
+const payloadAmounts = asObject({ nativeAmount: asString, networkFee: asString, tokenId: asOptional(asString, null) })
```

Comment:
Not quote. Since this is a payload from `accountbased`, we need to handle `tokenId === undefined` as a possible case for backwards compatibility. The core can't correct the payload since it's in `otherMethods`